backend/s390x.pm

```diff
@@ -69,15 +66,15 @@ sub do_stop_vm {
 
     # now cleanly disconnect from the guest and then kill the local
     # Xvnc
-    $self->deactivate_console({testapi_console => 'sut'});
-    $self->deactivate_console({testapi_console => 'worker'});
+    # FIXME: "console worker does not exist at /home/os-autoinst/consoles/s3270.pm line 491 thread 1."
+    #$self->deactivate_console({testapi_console => 'sut'});
+    #$self->deactivate_console({testapi_console => 'worker'});
     return;
 }
 
 sub status {
     my ($self) = @_;
-    # FIXME: do something useful here.
-    carp "status not implemented";
+    return 'shutdown';
```

Comment:
well, they return nothing - you return something pretty concrete. Perhaps you can track the status in the backend and set it to shutdown in stop_vm